pyproject.toml

```diff
@@ -21,13 +21,22 @@ homepage = "https://github.com/simon-weber/gpsoauth"
 repository = "https://github.com/simon-weber/gpsoauth"
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.6.1"
 pycryptodomex = "^3.10.1"
 requests = "^2.25.1"
 importlib-metadata = "^3.10.0"
 
 [tool.poetry.dev-dependencies]
+pre-commit = "^2.11.1"
+black = "^20.8b1"
+isort = "^5.8.0"
+pytest = "^6.2.2"
 
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.isort]
+profile = "black"
```

Comment:
ah, cool; I'm guessing this avoids the conflicts I've seen occasionally.

Reply:
Yes, some tools need to be configured to be compatible with `black` 🙂
https://black.readthedocs.io/en/stable/compatible_configs.html